doc/_includes/inverse.rst

```diff
@@ -74,15 +57,26 @@ The expected value of the current amplitudes at time *t* is then given by
 :math:`\hat{j}(t) = Mx(t)`, where :math:`x(t)` is a vector containing the
 measured MEG and EEG data values at time *t*.
 
+For computational convenience, the linear inverse operator is
+not computed explicitly. See :ref:`mne_solution` for mathematical
+details, and :ref:`CIHCFJEI` for a detailed example.
+
 .. _mne_regularization:
 
 Regularization
---------------
+~~~~~~~~~~~~~~
+
+The a priori variance of the currents is, in practice, unknown. We can express
+this by writing :math:`R' = R/ \lambda^2 = R \lambda^{-2}`, which yields the
+inverse operator
 
-The a priori variance of the currents is, in practise, unknown. We can express
-this by writing :math:`R' = R/ \lambda^2`, which yields the inverse operator
+.. math::
+   :label: inv_m
 
-.. math::    M = R G^T (G R G^T + \lambda^2 C)^{-1}\ ,
+    M &= R' G^T (G R' G^T + C)^{-1} \\
+      &= R \lambda^{-2} G^T (G R \lambda^{-2} G^T + C)^{-1} \\
+      &= R \lambda^{-2} G^T \lambda^2 (G R G^T + \lambda^2 C)^{-1} \\
+      &= R G^T (G R G^T + \lambda^2 C)^{-1}\ ,
 
 where the unknown current amplitude is now interpreted in terms of the
```

Comment:
around here I would explicit:

```
\min_j (y - Gj)^\top C^{-1} (y - Gj) + \lambda^2 j^\top R^{-1} j
```

you also see that transpose in latex should be written \top so it does not read like the lettre T that could use elsewhere (eg to denote time index)

Reply:
I don't know if you have a notation for the data so I used y here.